lib/builders/latexmk.js

```diff
@@ -33,13 +33,14 @@ export default class LatexmkBuilder extends Builder {
 
   constructArgs (filePath) {
     const outputFormat = atom.config.get('latex.outputFormat') || 'pdf'
+    const enableSynctex = atom.config.get('latex.enableSynctex') !== false
 
     const args = [
       '-interaction=nonstopmode',
       '-f',
       '-cd',
       `-${outputFormat}`,
-      '-synctex=1',
+      `-synctex=${enableSynctex ? 1 : 0}`,
```

Comment:
I think it would be nicer if this is enabled in a similar fashion to what we do for `enableShellEscape` below. I'll change how we apply the `outputFormat` option at some point to make everything consistent.